worlds/civ_6/Items.py

```diff
@@ -0,0 +1,302 @@
+        _items_by_civ_name = {item.civ_name: item for item in item_table.values() if item.civ_name}
+
+    item = _items_by_civ_name.get(item_name, None)
+    if not item:
+        raise Exception(f"Item {item_name} not found in item_table")
+    return item
+
+
+def _generate_tech_items(id_base: int, required_items: List[str], progressive_items: Dict[str, str]) -> Dict[str, CivVIItemData]:
+    # Generate Techs
+    existing_techs = get_existing_techs_data()
+    tech_table: Dict[str, CivVIItemData] = {}
+
+    tech_id = 0
+    for tech in existing_techs:
+        classification = ItemClassification.useful
+        name = tech["Name"]
+        civ_name = tech["Type"]
+        if civ_name in required_items:
+            classification = ItemClassification.progression
+        progression_name = None
+        check_type = CivVICheckType.TECH
+        if civ_name in progressive_items.keys():
+            progression_name = format_item_name(progressive_items[civ_name])
+
+        tech_table[name] = CivVIItemData(
+            name=name,
+            civ_vi_id=tech_id,
+            cost=tech["Cost"],
+            item_type=check_type,
+            id_offset=id_base,
+            classification=classification,
+            progression_name=progression_name,
+            civ_name=civ_name,
+            era=EraType(tech["EraType"])
+        )
+
+        tech_id += 1
+
+    return tech_table
+
+
+def _generate_civics_items(id_base: int, required_items: List[str], progressive_items: Dict[str, str]) -> Dict[str, CivVIItemData]:
+    civic_id = 0
+    civic_table: Dict[str, CivVIItemData] = {}
+    existing_civics = get_existing_civics_data()
+
+    for civic in existing_civics:
+        name = civic["Name"]
+        civ_name = civic["Type"]
+        progression_name = None
+        check_type = CivVICheckType.CIVIC
+
+        if civ_name in progressive_items.keys():
+            progression_name = format_item_name(progressive_items[civ_name])
+
+        classification = ItemClassification.useful
+        if civ_name in required_items:
+            classification = ItemClassification.progression
+
+        civic_table[name] = CivVIItemData(
+            name=name,
+            civ_vi_id=civic_id,
+            cost=civic["Cost"],
+            item_type=check_type,
+            id_offset=id_base,
+            classification=classification,
+            progression_name=progression_name,
+            civ_name=civ_name,
+            era=EraType(civic["EraType"])
+        )
+
+        civic_id += 1
+
+    return civic_table
+
+
+def _generate_progressive_district_items(id_base: int) -> Dict[str, CivVIItemData]:
+    progressive_table: Dict[str, CivVIItemData] = {}
+    progressive_id_base = 0
+    progressive_items = get_progressive_districts_data()
+    for item_name in progressive_items.keys():
+        classification = ItemClassification.progression
+        if item_name in NON_PROGRESSION_DISTRICTS:
+            classification = ItemClassification.useful

```

Comment:
```suggestion
        classification = ItemClassification.useful if item_name in NON_PROGRESSION_DISTRICTS else ItemClassification.progression
```
but also it's fine as-is